### PR TITLE
kubelb: datacenter kubeconfig should have precedence over seed kubeconfig

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
**What this PR does / why we need it**:
Datacenter-level configuration for KubeLB management cluster should take precedence over the seed configurations.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

/kind bug

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

**Comment by ahmedwaleedmalik** [comment]
> pre-kubermatic-lint 

`rewrite if-else to switch statement` 😒

**Comment by ahmedwaleedmalik** [comment]
cc @kubermatic/sig-release-managers

File: pkg/ee/kubelb/controller.go [hunk 1]
```diff
@@ -412,15 +412,16 @@ func (r *reconciler) getKubeLBManagementClusterClient(ctx context.Context, seed
...
+	switch {
+	case dc.Spec.KubeLB != nil && dc.Spec.KubeLB.Kubeconfig.Name != "":
 		name = dc.Spec.KubeLB.Kubeconfig.Name
 		namespace = dc.Spec.KubeLB.Kubeconfig.Namespace
-	} else {
+	case seed.Spec.KubeLB != nil && seed.Spec.KubeLB.Kubeconfig.Name != "":
 		name = seed.Spec.KubeLB.Kubeconfig.Name
 		namespace = seed.Spec.KubeLB.Kubeconfig.Namespace
+	default:
+		return nil, fmt.Errorf("kubeLB management kubeconfig not found")
```
Comment (xrstf) [inline pkg/ee/kubelb/controller.go:424]: `errors.New("neither Seed nor datacenter have a KubeLB kubeconfig configured")`

**Pull request merged**